cli/src/commands/auth.rs

```diff
@@ -27,12 +27,34 @@ async fn handle_auth_login(mut config: Config, config_path: &Path) -> CommandRes
 }
 
 /// Display the current authentication status to the user.
-pub fn handle_auth_status(config: &Config) {
-    if config.auth_info.offline_access.is_some() {
-        print_user_success!("Currently authenticated with long lived refresh token");
-    } else {
-        print_user_warning!("User is not currently authenticated");
+pub async fn handle_auth_status(
+    mut config: Config,
+    timeout: Option<u64>,
+    ignore_certs: bool,
+) -> CommandResult {
+    // Create a client with our auth token attached.
+    let api = PhylumApi::new(
+        &mut config.auth_info,
+        &config.connection.uri,
+        timeout,
+        ignore_certs,
+    )
+    .await
+    .map_err(|err| anyhow!("Error creating client").context(err))?;
```

Comment:
The error and context are backwards here. `err` is the error and `"Error creating client"` should be the context.

```suggestion
    .map_err(|err| err.context("Error creating client").into())?;
```

Reply:
I actually just copied that from `phylum.rs`, so then it's probably wrong there too.